rllib/examples/multi_agent_two_trainers.py

```diff
@@ -77,29 +78,7 @@ def select_policy(algorithm, framework):
         else:
             raise ValueError("Unknown algorithm: ", algorithm)
 
-    # You can also have multiple policies per algorithm, but here we just
```

Comment:
I had to reorder the configuration in this file a little.
It is the same configuration, only that `policies` now also contains the configs, instead of empty dicts.
This makes it so that the configs end up in the policies instead of them being fed with default configs.

Reply:
So..

```
"ppo_policy": (
            select_policy("PPO", args.framework),
            obs_space,
            act_space,
            {},
        ),
```
        
 because
 
```
 "ppo_policy": (
            select_policy("PPO", args.framework),
            obs_space,
            act_space,
            ppo_config,
        ),
```